sr25519-java/src/lib.rs

```diff
@@ -116,7 +105,7 @@ pub unsafe extern "system" fn Java_jp_co_soramitsu_fearless_1utils_encrypt_Sr255
  * @return true if signature is valid, false otherwise
  */
 #[no_mangle]
-pub unsafe extern "system" fn Java_jp_co_soramitsu_fearless_1utils_encrypt_Sr25519_verify(
+pub unsafe extern "system" fn Java_io_novasama_substrate_1sdk_1android_encrypt_Sr25519_verify(
```

Comment:
`_1` is needed as per JNI excaping spec:
![image](https://user-images.githubusercontent.com/70131744/226574431-96fc3eab-5fe4-4e06-9d0d-90e2185c4109.png)